tests/layer_tests/tensorflow_tests/test_tf_RandomUniform.py

```diff
@@ -90,7 +90,7 @@ def create_tf_random_uniform_net(self, global_seed, op_seed, x_shape, min_val, m
     @pytest.mark.nightly
     @pytest.mark.precommit
     @pytest.mark.precommit_tf_fe
-    @pytest.mark.xfail(condition=platform.system() == 'Darwin' and platform.machine() == 'arm64',
+    @pytest.mark.xfail(platform.machine() in ["aarch64", "arm64", "ARM64"],
```

Comment:
why this change needed?

Reply:
Who will address this limitation in the future?

Reply:
I have a conversation with the team that supports these tests, there is no way to force f16 inference precision in these tests. It was suggested to skip these tests on arm64 platform.
As soon as inference precision option is added to tests, skip should be removed.